.eslintrc.js

```diff
@@ -34,9 +34,11 @@ module.exports = {
 
   overrides: [
     {
-      files: 'test/**/*.js',
-      env: {
-        jest: true
```

Comment:
I removed this because it's no longer necessary, as all the test files are now in TypeScript, and TypeScript will make sure the Jest globals are used properly